src/search.cpp

```diff
@@ -1338,7 +1337,7 @@ namespace {
     (ss+1)->ply = ss->ply + 1;
     bestMove = MOVE_NONE;
     inCheck = pos.checkers();
-    priorCapture = pos.captured_piece();
+    priorCapture = !!pos.captured_piece();
```

Comment:
bool(pos.captured_piece()) is what we use throughout the code for this.

Reply:
Yes, that's indeed better, thanks! I'll update the PR shortly...